Vagrantfile

```diff
@@ -0,0 +1,77 @@
+# vim: set ft=ruby sw=2 ts=2:
+# -*- mode: ruby -*-
+
+DB_IP  = '192.168.3.42'
+WEB_IP = '192.168.3.43'
+
+DB_HOSTNAME  = 'bmo-db.vm'
+WEB_HOSTNAME = 'bmo-web.vm'
+
+# this is for centos 6 / el 6
+VENDOR_BUNDLE_URL = 'https://moz-devservices-bmocartons.s3.amazonaws.com/bmo/vendor.tar.gz'
+
+# All Vagrant configuration is done below. The '2' in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure('2') do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  config.vm.provision 'ansible_local', run: 'always' do |ansible|
+    ansible.playbook = 'vagrant_support/playbook.yml'
+    ansible.extra_vars = {
+      WEB_IP:            WEB_IP,
+      DB_IP:             DB_IP,
+      WEB_HOSTNAME:      WEB_HOSTNAME,
+      DB_HOSTNAME:       DB_HOSTNAME,
+      VENDOR_BUNDLE_URL: VENDOR_BUNDLE_URL,
+    }
+  end
+
+  config.vm.define 'db' do |db|
+    db.vm.box = 'centos/6'
+    db.vm.hostname = DB_HOSTNAME
+    db.vm.network 'private_network', ip: DB_IP
```

Comment:
It would be good to add `db.vm.network :forwarded_port, guest: 22, host: 12914, id: 'ssh'` as people might have other boxes running in parallel, which might cause conflict on port.